packages/editor/src/panels/materials/helpers.tsx

```diff
@@ -91,7 +91,7 @@ export async function saveMaterial(sourcePath: string) {
   const gltf = (await exportMaterialsGLTF([UUIDComponent.getEntityByUUID(materialUUID)], {
     binary: false,
     relativePath,
-    projectName: projectName
```

Comment:
This seemed to have been ported out of this PR since this never got merged. But I kept my change anyway since it's a little cleaner